hyperspy/model.py

```diff
@@ -27,8 +27,10 @@
 import scipy
 import scipy.odr as odr
 from scipy.optimize import (leastsq, least_squares,
-                            minimize, differential_evolution)
+                            minimize, differential_evolution, nnls)
 from scipy.linalg import svd
+from numpy.linalg import lstsq as linear
```

Comment:
instead of numpy.linalg, why not use SciPy's version which *does* support bounds? 

```
from scipy.optimize import (leastsq, least_squares,
                            minimize, differential_evolution, nnls, lsq_linear)
```

[Reference](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.lsq_linear.html).